build.sbt

```diff
@@ -31,6 +31,15 @@ lazy val core = crossProject(JVMPlatform, JSPlatform, NativePlatform).in(file("c
       "org.scalacheck" %%% "scalacheck" % "1.13.5" % "test")
   ).
   jsSettings(commonJsSettings: _*).
+  nativeSettings(
+    crossScalaVersions := Seq(Scala211),
+    // Don't compile shared/src/test
+    unmanagedSourceDirectories.in(Test) := Seq(sourceDirectory.in(Test).value),
+    test := Def.taskDyn {
```

Comment:
Note that utest is published for native. These workarounds should not be necessary in the future if scalacheck/scalatest support native.